src/main/java/Duke.java

```diff
@@ -1,10 +1,29 @@
+
```

Comment:
I noticed multiple instances of incorrect import formatting.

Reply:
Done!